### ISSUE TITLE
Add option for Swype style double letters

### ISSUE THREAD

**Issue by Stealthstarcat** [futo-org/android-keyboard#250]
Referring to the original Swype keyboard on the Galaxy S. Drawing a small circle over a letter types two of that letter. On FUTO it thinks I mean an adjacent letter though I never leave the singular letter. A toggle for this would be nice.

**Comment by ocouch** [comment]
Agree, makes it difficult to use swipe typing as the accuracy of word prediction is very poor

**Comment by NicoWeio** [comment]
Just to make sure I understood the problem right:
- You try to double-tap a letter
- Your touch input is recognized as a swipe over a small distance instead
- You want the keyboard to not interpret this as a swipe, but as the double-tap you intended

**Comment by ocouch** [comment]
NicoQeio no, maybe you tried tunning that through a translator? Not sure how you got so lost....Easiest to just show you with a video:

https://www.youtube.com/watch?v=kudXp5Zk9uU

**Comment by abb128** [comment]
The algorithm in 0.1.29 does a better job of understanding a circle as a double letter
